numpy/typing/_dynamic_types.py

```diff
@@ -0,0 +1,51 @@
+"""
```

Comment:
Can we call this `_type_aliases.py` or `_platform_aliases.py` instead?

Reply:
Renamed as of https://github.com/numpy/numpy/pull/17514/commits/59d730e5f674e75859e2dc322802d9f126a32173.